genesyscloud/telephony_providers_edges_phonebasesettings/resource_genesyscloud_telephony_providers_edges_phonebasesettings_utils.go

```diff
@@ -138,7 +138,7 @@ func BuildTelephonyLineBaseProperties(d *schema.ResourceData) *map[string]interf
 		lineBaseMap := lineBase[0].(map[string]interface{})
 
 		properties := map[string]interface{}{
-			"station_persistent_enabled": &map[string]interface{}{
+			"station_persistent_webrtc_enabled": &map[string]interface{}{
```

Comment:
There are both "station_persistent_enabled" and "station_persistent_webrtc_enabled" attributes returning from linebasesettings:
```
╰─ gc telephony providers edges linebasesettings list -a -p ps_japan | grep 'station_persistent' | sort | uniq                                                                                                                               ─╯
      "station_persistent_enabled": {
      "station_persistent_timeout": {
      "station_persistent_webrtc_enabled": {
```

Reply:
The original purpose of `station_persistent_enabled` was to control `station_persistent_webrtc_enabled` (issue #1360 explains this) 

I added a new field for this instead just incase anyone was using `station_persistent_enabled` to actually control `station_persistent_enabled`